lib/ecto/migrator.ex

```diff
@@ -93,6 +93,10 @@ defmodule Ecto.Migrator do
           If this can be an issue, we recommend to rollback #{version} and change \
           it to a version later than #{latest}.
           """
+
+          if opts[:strict_version_order] do
+            raise Ecto.MigrationError
+          end
```

Comment:
Can you please update the docs for this function to mention the `:strict_version_order` option?

Reply:
Also, what do you think of including the message in the exception? Something like this:

```elixir
message = """
...
"""

if opts[:strict_version_order] do
  raise Ecto.MigrationError, message
else
  Logger.warn(message)
end
```

Reply:
Nice idea. Updated.